BaseFilter/sections/specific.txt

```diff
@@ -566,6 +566,13 @@ $subdocument,third-party,denyallow=disqus.com,domain=welovemanga.one
 ! NOTE: Specific rules
 !
 !
+ww2-weapons.com###sgpb-popup-dialog-main-div-wrapper
+ww2-weapons.com##.sgpb-theme-6-overlay
```

Comment:
I'd prefer to use this class, if the site has themes, and the number may be changed:
![image](https://github.com/user-attachments/assets/4f6116fe-89d2-4c4b-a33d-db3768b3d7d5)